push-messaging-and-notifications/main.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+var API_KEY = 'AIzaSyBBh4ddPa96rQQNxqiq_qQj7sq1JdsNQUQ';
+
+var isPushEnabled = false;
+
+function sendToServer(subscriptionId, endpoint) {
+  // The curl command to trigger a push message straight from GCM
+  var curlCommand = 'curl --header "Authorization: key=' + API_KEY +
+    '" --header Content-Type:"application/json" ' + endpoint + 
+    ' -d "{\\"registration_ids\\":[\\"' + subscriptionId + '\\"]}"';
+
+  // Below is a curl command to test sending a push message
+  console.log(curlCommand);
```

Comment:
This info, and a lot of other messages that get logged, would be useful to display on page, so that folks who don't have the JS console open will see it. Could you add in a helper method to append text to a visible element and use that instead/in addition to the `console` logging?